packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -351,39 +365,47 @@ export default decorate([
 
         let newLog
         log.tasks.forEach((task, key) => {
-          if (task.tasks === undefined || get(() => task.data.type) !== 'VM') {
+          if (get(() => task.data.type) !== 'VM') {
```

Comment:
Why this change?

Reply:
`VM` sub tasks can be undefined in case of an error on the `VM` level.

Reply:
It was handled before, this change is completely unrelated to this PR.

Reply:
But, we should also be able to search for failure VMs which doesn't have tasks.